ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/ForkIdManager.java

```diff
@@ -61,6 +64,9 @@ public ForkIdManager(final Blockchain blockchain, final List<Long> nonFilteredFo
   }
 
   public ForkId computeForkId() {
+    if (legacyEth64) {
+      return forkAndHashList.size() > 0 ? forkAndHashList.get(forkAndHashList.size() - 1) : null;
```

Comment:
If you use `isEmpty` here it's easier to parse at a glance how it differs from the version below

Reply:
I agree. Done